WcaOnRails/app/helpers/notifications_helper.rb

```diff
@@ -67,6 +67,15 @@ def notifications_for_user(user)
       }
     end
 
+    user.delegated_competitions.visible.over
```

Comment:
```
mysql> SELECT name, end_date, isConfirmed, showAtAll FROM Competitions WHERE showAtAll = false AND end_date < CURDATE();
+-------------------------+------------+-------------+-----------+
| name                    | end_date   | isConfirmed | showAtAll |
+-------------------------+------------+-------------+-----------+
| BTE Rubik's Day 2017    | 2017-05-13 |           0 |         0 |
| Florida Spring 2017     | 2017-03-25 |           0 |         0 |
| IUST Cube 2017          | 2017-04-20 |           0 |         0 |
|  SLS Czestochowa 2017   | 2017-01-08 |           0 |         0 |
| Virginia Open Fall 2016 | 2016-10-08 |           0 |         0 |
| Yale Fall 2016          | 2016-10-15 |           0 |         0 |
+-------------------------+------------+-------------+-----------+
6 rows in set (0.00 sec)
```

I would just find it kinda weird to get notification asking to submit a report where there was no actual competition at all. @jfly any opinion on that?